assets/styles/style.css

```diff
@@ -152,3 +177,12 @@ iframe {
 .is-animated .beer-foam {
     animation: foam-show 3000ms infinite
 }
+
+.jumbotron.footer {
+    margin-top: 80px;
+    height: 110px;
```

Comment:
@mdvb1001 I think you may want `bottom: 0;` here. It looks like there is a gap below the footer.